typescript/cli/src/config/warp.ts

```diff
@@ -147,6 +153,12 @@ export async function createWarpRouteDeployConfig({
         message: `Could not retrieve mailbox address from the registry for chain "${chain}". Please enter a valid mailbox address:`,
       }));
 
+    const proxyAdmin: DeployedOwnableConfig | undefined =
```

Comment:
Imo, our commands should always be config driven I.e config should match the intention. Here, we are deploying a proxy admin but the proxy admin config is undefined. 

If you want to deploy a warp route with an existing proxy admin, then proxyAdmin.address should be set to something. If it doesn’t exist, then assume we’re doing a new proxyAdmin deployment.

this allows you to potentially update the existing proxyAdmin to another existing proxyAdmin just by comparing the expected vs actual proxyAdmin.address